### ISSUE TITLE
Does not work with ASP.NET Core 2

### ISSUE THREAD

**Issue by mbp** [mattfrear/Swashbuckle.AspNetCore.Filters#51]
When using Swashbuckle.AspNetCore.Filters 3.0.2 with an ASP.NET Core 2 application, you get the following exception:

```
MissingMethodException:
Method not found: 'System.IServiceProvider Microsoft.Extensions.DependencyInjection.ServiceCollectionContainerBuilderExtensions.BuildServiceProvider(Microsoft.Extensions.DependencyInjection.IServiceCollection)'.

Swashbuckle.AspNetCore.Filters.SwaggerGenOptionsExtensions.AddSwaggerExamples(SwaggerGenOptions swaggerGenOptions, IServiceCollection services)
```

It would appear the issue is with the `Microsoft.Extensions.DependencyInjection` dependency (1.1.0) which made a breaking change to return `ServiceProvider` instead of `IServiceProvider` in the `BuildServiceProvider()` method.

**Comment by veler** [comment]
Hello there =) I'm facing the same issue this morning. Looking forward for a fix. Thanks =)

**Comment by mattfrear** [comment]
OK, I've fixed it locally. Release coming later today.

**Comment by mattfrear** [comment]
Fixed in 3.0.3 https://www.nuget.org/packages/Swashbuckle.AspNetCore.Filters/3.0.3

You'll need to change your Startup.cs

```
services.AddSwaggerGen(c =>
{
    c.AddSwaggerExamples(services.BuildServiceProvider());
```
